Shootingat Resolved Project/Assets/Scripts/Characters/Enemies/EnemyShootBasic.cs

```diff
@@ -16,6 +16,9 @@ public class EnemyShootBasic : EnemyFollow
         [SerializeField] private float minDistanceAwayFromPlayer;
         [SerializeField] private float shootDst;
 
+        [SerializeField] private int bulletDamage;
```

Comment:
Needs to be refactored. This was just to fix the bug of shooting with player stats.